docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+
+services:
+  pihole:
+    container_name: pihole
+    image: scantineau/pihole:latest
```

Comment:
Does this mean you will have to update the repository for the changes to be pushed into the docker image on the hub? Because in this case I would probably look into submitting it myself